.ci/packaging.groovy

```diff
@@ -154,6 +155,31 @@ pipeline {
                   prepareE2ETestForPackage("${BEATS_FOLDER}")
                 }
               }
+              stage('Package Docker images for linux/arm64'){
+                agent { label 'arm' }
```

Comment:
Since docker login is required in ARM then we need to use a specific worker for the time being:

```suggestion
                agent { label 'worker-0eaeecf32aca4fe3a' } // TODO: use 'arm' once the problem has been fixed in all the ARM workers.
```

For instance, golang-crossbuild uses the above label, to be changed in the future (see - https://github.com/elastic/golang-crossbuild/pull/77)